src/clients/remote.rb

```diff
@@ -56,8 +56,6 @@ def main
         ),
         "id"         => "remote",
         "guihandler" => fun_ref(method(:RemoteGUI), "any ()"),
-        "initialize" => fun_ref(Remote.method(:Read), "boolean ()"),
-        "finish"     => fun_ref(Remote.method(:Write), "boolean ()"),
         "actions"    => {
           "list"  => {
```

Comment:
Does the `list` command still work properly? The initialization has been moved from the global section only to the `allow` command. The question is do the other commands still work when the initialization is skipped?